src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java

```diff
@@ -4,17 +4,25 @@
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.logging.PageObjectLogging;
-
+import com.wikia.webdriver.pageobjectsfactory.componentobject.activity.Activity;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.activity.ActivityPageFactory;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.activity.creators.*;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.FindBys;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 public class SpecialWikiActivityPageObject extends SpecialPageObject {
 
+  @FindBys(@FindBy(css = "li[class*=\"activity-type\"]"))
+  private List<WebElement> activityWebElementList;
+  @FindBys(@FindBy(css = "li.activity-ns-0"))
+  private List<WebElement> activitiesList; //includes all of edit, new etc.
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)